### ISSUE TITLE
[flow] No named export called `change`

### ISSUE THREAD

**Issue by Pegase745** [erikras/redux-form#3214]
### Are you submitting a **bug report** or a **feature request**?
Possibly a bad export of action types

### What is the current behavior?
`change` action is working fine, the only problem is that Flow throws the following
```bash
  4: import { change } from 'redux-form';
              ^^^^^^ Named import from module `redux-form`. This module has no named export called `change`.
```

### What is the expected behavior?
No flow errors

### What's your environment?

Redux-form 7.0.1, Node 4.7.2, Flow 0.50.0

**Comment by DimitryDushkin** [comment]
Also there is no `input` in `FieldProps` flow type declaration.

**Comment by erikras** [comment]
Oops. `FieldProps` are the props that `Field` ___accepts___, not provides.

The docs are correct, but the exported type is wrong. Will fix. Thanks, @DimitryDushkin.

**Comment by bodyflex** [comment]
`submit` is also not exported according to flow

**Comment by erikras** [comment]
Published fix in [`v7.0.2`](https://github.com/erikras/redux-form/releases/tag/v7.0.2).

**Comment by alex-shamshurin** [comment]
Please reopen this issue, it still hap problems in `import { submit } from 'redux-form/es/actions'`
